ACPs/151-icm-use-current-block-pchain-height/README.md

```diff
@@ -0,0 +1,38 @@
+| ACP           | 151                                                             |
+| :------------ | :-------------------------------------------------------------- |
+| **Title**     | Use current block P-Chain height for ICM signature verification |
+| **Author(s)** | Ian Suvak ([@iansuvak](https://github.com/iansuvak))            |
+| **Status**    | Proposed                                                        |
+| **Track**     | Standards                                                       |
+
+## Abstract
+
+Proposes that VMs use the P-Chain height of the current block being built instead of the parent block when verifying aggregated signatures of Avalanche Interchain Messages (ICM). This will allow for a more reliable way to determine which validators should participate in signing the message, and remove unnecessary waiting periods.
```

Comment:
```suggestion
Proposes that the ProposerVM uses the P-Chain height of the current block being built instead of the parent block when verifying aggregated signatures of Avalanche Interchain Messages (ICM). This will allow for a more reliable way to determine which validators should participate in signing the message, and remove unnecessary waiting periods.
```

Reply:
ProposerVM in this case just passes in the context but doesn't do the verification itself, right? The real proposal here is that ProposerVM *passes* the current block's p-chain height as the context for verification of state to the inner VM so that the inner VM can use that for ICM verification. 

ProposerVM doesn't control what the inner VM does with the context once it's passed in right. I'm not sure what is the best way to word this though.

Reply:
Ah yeah, fair enough!

Technically this is only a change to the ProposerVM, which I was trying to get across here, but that of course impacts any VMs that use the ProposerVM

Reply:
How about that updated suggestion?